src/renderer/components/helpers/ChatMethods.ts

```diff
@@ -217,7 +217,7 @@ export async function createChatByContactIdAndSelectIt(
   )
 
   if (!chatId) {
-    throw new Error(window.static_translate('create_chat_error_desktop'))
+    throw new Error('could not create chat')
   }
```

Comment:
Anyways thinking about it this if statement is a remnant of using the cffi, it is useless as createChatByContactId does not return `0`/`null`, it's return type is `Result<u32>`, so we can remove the whole if statement here.

Reply:
even  better!

Reply:
made a pr for it https://github.com/deltachat/deltachat-desktop/pull/3475